plugins/eos/gs-plugin-eos.c

```diff
@@ -1024,32 +1017,13 @@ add_updates (GsPlugin *plugin,
 						"com.endlessm.quote_of_the_day.en.desktop",
 						"com.endlessm.word_of_the_day.en.desktop",
 						NULL};
-	const char *hack_proxied_apps[] = {"com.endlessm.Clubhouse.desktop",
-					   "com.endlessm.Fizzics.desktop",
-					   "com.endlessm.GameStateService.desktop",
-					   "com.endlessm.HackAccountService.desktop",
-					   "com.endlessm.HackComponents.desktop",
-					   "com.endlessm.HackSoundServer.desktop",
-					   "com.endlessm.HackToolbox.desktop",
-					   "com.endlessm.HackUnlock.desktop",
-					   "com.endlessm.Hackdex_chapter_one.desktop",
-					   "com.endlessm.Hackdex_chapter_two.desktop",
-					   "com.endlessm.LightSpeed.desktop",
-					   "com.endlessm.OperatingSystemApp.desktop",
-					   "com.endlessm.Sidetrack.desktop",
-					   NULL};
 
 	GsPluginData *priv = gs_plugin_get_data (plugin);
-	gboolean is_hack_product = g_strcmp0 (priv->product_name, "hack") == 0;
 
 	process_proxy_updates (plugin, list,
 			       framework_proxy_app,
 			       FALSE,
```

Comment:
This parameter is now always `FALSE` so should be removed along with the code for the `TRUE` path.

Reply:
Yes, thought about it but decided to not remove this code as I thought this codepath could be useful (although currently unused) in case we keep using proxy apps.

I can still remove it if you prefer but I think this is independent of the fixes on this PR.

Reply:
It was introduced as a last-minute hack to install new Hack 1 apps only on Hack 1 systems. We should remove it.

Reply:
Done.